CHANGES.rst

```diff
@@ -1,3 +1,7 @@
+Release 0.0.7
+=============
+* **datasets.fetch_employee_salaries**: change the origin of download for employee_salaries.
```

Comment:
Please also mention that this changed the function signature: the function now return a Bunch with a dataframe under the field "data", and not the path to the csv file. Also, the field "description" has been renamed to "DESCR".

Mentioning this is important, because these changes can break user code, and they need to quickly identify them from the changelog.